docs/deploy-event-router.md

```diff
@@ -0,0 +1,111 @@
+# Deploying Event Router
+As of 4.0, the EventRouter is not deployable by the `cluster-logging-operator`. Creating a deployment using the following template
+will enable event collection equivalent to the same capability from 3.11 releases or earlier.
+Currently, all events are logged to `STDOUT`, collected by the logging collector and indexed to the `.operations` index when 
+the `EventRouter` is deployed into an operations namespace (e.g. `openshift-*`). Elevated permissions are required to create the cluster level roles and bindings referenced in the following template:
+
+```
+kind: Template
+apiVersion: v1
+metadata:
+  name: eventrouter-template
+  annotations:
+    description: "A pod forwarding kubernetes events to cluster logging stack."
+    tags: "events,EFK,logging, cluster-logging"
+objects:
+  - kind: ServiceAccount
+    apiVersion: v1
+    metadata:
+      name: cluster-logging-eventrouter
+      namespace: ${NAMESPACE}
+  - kind: ClusterRole
+    apiVersion: v1
+    metadata:
+      name: event-reader
+    rules:
+    - apiGroups: [""]
+      resources: ["events"]
+      verbs: ["get", "watch", "list"]
+  - kind: ClusterRoleBinding
+    apiVersion: v1
+    metadata:
+      name: event-reader-binding
+    subjects:
+    - kind: ServiceAccount
+      name: cluster-logging-eventrouter
+      namespace: ${NAMESPACE}
+    roleRef:
+      kind: ClusterRole
+      name: event-reader
+  - kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: cluster-logging-eventrouter
+      namespace: ${NAMESPACE}
+    data:
+      config.json: |-
+        {
+          "sink": "stdout"
+        }
+  - kind: Deployment
+    apiVersion: apps/v1
+    metadata:
+      name: cluster-logging-eventrouter
+      namespace: ${NAMESPACE}
+      labels:
+        component: eventrouter
+        logging-infra: eventrouter
+        provider: openshift
+    spec:
+      selector:
+		matchLabels:
+          component: eventrouter
+          logging-infra: eventrouter
+          provider: openshift
+      replicas: "1"
```

Comment:
@jcantrill 
Should this be `replicas: 1` withouth the quotes? I was getting a `unrecognized type: int32` error until I removed the quotes. A lucky guess based on https://bugzilla.redhat.com/show_bug.cgi?id=1595538#c19